ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriter.java

```diff
@@ -98,12 +104,24 @@ public void write(final OutputStream out) {
     }
   }
 
-  private void write(final OutputStream output, final GenericRow row) throws IOException {
-    objectMapper.writeValue(output, StreamedRow.row(row));
+  private void write(final OutputStream output, final StreamedRow row) throws IOException {
+    objectMapper.writeValue(output, row);
     output.write("\n".getBytes(StandardCharsets.UTF_8));
     output.flush();
   }
 
+  private StreamedRow buildHeader() {
+    // Push queries only return value columns, but query metadata schema includes key and meta:
```

Comment:
With the comment, you mean you want the schema to only contain the value columns, but currently they return also the meta columns and that's what you are working on fixing?

Reply:
Nope. The schema in the query metadata contains key and meta columns. But push queries only return value columns.  Hence in this method I create a new schema with only the value columns.

The longer term fix is to store the right schema in the query metadata, but that's outside the scope of this PR.

The outstanding issue with schemas is for _pull_ queries, not _push_.  Pull queries return key columns, but the returned schema currently doesn't include this.